packages/components/popper/src/content.vue

```diff
@@ -84,7 +84,7 @@ const {
 })
 
 const formItemContext = inject(formItemContextKey, undefined)
-const arrowOffset = ref<number>()
+const arrowOffset = ref<number>(props.arrowOffset)
```

Comment:
Here we may add a watch `props.arrowOffset` to match the dynamic changes of `props.arrowOffset`.